### PR TITLE
Cherry-pick PR #103 to develop

### PR DESCRIPTION
**Pull request by niladrih**
Cherry-pick fb6ae57d1c995f5020173cb6291b2f19162303ef into develop and fix the label used in kubectl-mayastor to query helm chart release name, during `kubectl mayastor install upgrade-operator`.

### CODE REVIEW

**Comment by niladrih** [comment]
bors try

**Review by tiagolobocastro — APPROVED** [review]

File: k8s/plugin/src/resources/upgrade.rs [hunk 1]
```diff
@@ -549,7 +549,7 @@ pub async fn get_release_name(ns: &str) -> Result<String, Error> {
     match get_deployment_for_rest(ns).await {
         Ok(deployments) => match deployments.items.get(0) {
             Some(deployment) => match &deployment.metadata.labels {
-                Some(label) => match label.get("helm-release") {
+                Some(label) => match label.get("openebs.io/release") {
```
[the remark this answers is not in the document]
Comment (niladrih) [reply k8s/plugin/src/resources/upgrade.rs:552]: Which check? bors?

[the remark this answers is not in the document]
Comment (niladrih) [reply k8s/plugin/src/resources/upgrade.rs:552]: If you're asking if the case matching will fail for a different value, then yes, likely so. This is not configurable. The label is hard-coded. On failure, it seems to be using a default value for the release name... this is likely 'mayastor'.

**Review by avishnu — APPROVED** [review]
Lgtm

**Comment by niladrih** [comment]
bors merge

**Pull request merged**